core/templates/dev/head/App.ts

```diff
@@ -15,6 +15,25 @@
 /**
  * @fileoverview Initialization and basic configuration for the Oppia module.
  */
+<<<<<<< HEAD
```

Comment:
Resolve the conflicts.

Reply:
I don't know where I need to add the modules created so I don't know how to fix this.